File: C++/414. Third Maximum Number.cpp
class Solution {
public:
    int thirdMax(vector<int>& n) {
        int m=1;
        sort(n.rbegin(),n.rend());
        for(int i=1;i<n.size();i++){
            if(n[i-1]!=n[i])
            m++;
            if(m==3)
            return n[i];
        }
        return n[0];
    }
};

File: JAVA/414. Third Maximum Number.java
class Solution {
    public int thirdMax(int[] n) {
        int m=1;
        Arrays.sort(n);
        for(int i=n.length-2;i>=0;i--){
            if(n[i+1]!=n[i])
            m++;
            if(m==3)
            return n[i];
        }
        return n[n.length-1];
    }
}